ffi/athcon/include/athcon/athcon.h

```diff
@@ -187,6 +187,19 @@ extern "C"
    */
   typedef athcon_address (*athcon_spawn_fn)(struct athcon_host_context *context, const uint8_t *blob, size_t blob_size);
 
+    /**
+   * Deploy program callback function.
+   *
+   * This callback function is used by a VM to deploy a new program template
+   *
+   * @param context   The pointer to the Host execution context.
+   * @param blob      The program's template (serialized code).
```

Comment:
```suggestion
   * @param blob      The program's template (bytecode).
```

"serialized code" sounds kinda funny